bin/symfony_requirements

```diff
@@ -43,9 +43,9 @@ foreach ($symfonyRequirements->getRecommendations() as $req) {
 }
 
 if ($checkPassed) {
-    echo_block('success', 'OK', 'Your system is ready to run Symfony2 projects');
```

Comment:
While you’re at it: you could update `/** @var $req Requirement */` to the more common variant `/** @var Requirement $req */` if you don’t mind